test/e2e/density.go

```diff
@@ -190,8 +190,10 @@ var _ = Describe("Density [Skipped]", func() {
 	for _, testArg := range densityTests {
 		name := fmt.Sprintf("should allow starting %d pods per node", testArg.podsPerNode)
 		if testArg.podsPerNode == 30 {
-			name = "[Performance] " + name
+			name = "[Feature:Performance] " + name
 			framework.addonResourceConstraints = density30AddonResourceVerifier()
+		} else {
+			name = "[Skipped] " + name
```

Comment:
> We can change Skipped to Manual to better reflect the meaning. WDYT?

I'd rather avoid manual running of tests as something that's expected.

> as they either don't provide any value (3)

Why do we still have the (3) test at all then?

> or are above our current SLO for density (100)

How about `[Feature:ExperimentalPerformance]` or something like that?  That sets the expectation that you all might eventually run this in an 'experimental' job that isn't expected to be stable, or something.